core/domain/feedback_services.py

```diff
@@ -651,14 +729,19 @@ def get_multiple_threads(thread_ids):
     Returns:
         list(FeedbackThread). The list of feedback threads.
     """
+    feedback_thread_models_with_none = (
+        feedback_models.GeneralFeedbackThreadModel.get_multi(thread_ids)
+    )
```

Comment:
This variable seems unnecessary? Can't we inline the function call below like it was originally?

Reply:
removed!